integrations/pagerDuty.js

```diff
@@ -281,15 +296,21 @@ async function resolveIncident(incidentId, email) {
  * @param {string} email      The current user's email
  */
 async function escalateIncident(incidentId, email) {
-  const { level } = await ux.prompt(escalatePrompt)
+  let chosenLevel = await ux.prompt(escalatePrompt)
+  while (chosenLevel.level === '' || isNaN(chosenLevel.level)) {
```

Comment:
The validate function from before used to do some funky regex. 

Not sure if it was correct but this won't ensure it's an integer. For example, 0.123 would still pass. Does it need to be an integer?

Reply:
It does, the escalation levels goes up by single digits as each one represents an user/schedule to notify/follow